File: 0001_0100/0025/0025.c
#include <stdio.h>

int main()
{
	FILE *f = fopen("input.txt", "r"), *q = fopen("output.txt", "w");
	int a, b;

	fscanf(f, "%d%d", &a, &b);
	fprintf(q, "%c", (a > b) ? '>' : (a < b) ? '<' : '=');
	return 0;
}

File: 0001_0100/0025/0025.go
package main

import (
	"fmt"
	"os"
)

func main() {
	f, _ := os.Open("input.txt")
	q, _ := os.Create("output.txt")
	var a, b int
	s := "="
	fmt.Fscan(f, &a, &b)
	if a > b {
		s = ">"
	} else if a < b {
		s = "<"
	}
	fmt.Fprint(q, s)
}
